tasks/steps/stop_instance.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: 'Stop and disable instance systemd service'
+  systemd:
+    name: '{{ instance_info.systemd_service }}'
+    state: stopped
+    enabled: false
+  tags: cartridge-instances
```

Comment:
Let's use `when: stopped`, similarly to restart task. By default this variable can be equal to `true`